tutorials/0. Theory.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Add some comment in the beginning of the paragraph explaining why it is a good idea to revisit Galerkin-method. Something like</p><p> </p><p>From mathematical standpoint, <code>PyDEns</code> is based on <code>DeepGalerkin</code>-algorithm. Unsurprisingly, the method is based on age-old <code>Galerkin</code> method. Let's first recall the essence of the method...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>